configs/envoy_service_to_service_v2.template.yaml

```diff
@@ -0,0 +1,316 @@
+{% import 'routing_helper_v2.template.yaml' as helper -%}
+{% import 'access_log_format_helper_v2.template.yaml' as access_log_helper -%}
+{% macro ingress_listener(protocol, address, port_value) -%}
+- name: listener_0
+  address:
+    socket_address:
+      protocol: {{protocol}}
+      address: {{address}}
+      port_value: {{port_value}}
+  filter_chains:
+  - filters:
+    - name: envoy.http_connection_manager
+      config:
+        codec_type: AUTO
+        stat_prefix: ingress_http
+        route_config:
+          name: local_route
+          virtual_hosts:
+          - name: local_service
+            domains:
+            - "*"
+            routes:
+            - match:
+                prefix: "/"
+                headers:
+                - name: content-type
+                  value: application/grpc
+              route:
+                cluster: main_website
+            - match:
+                prefix: "/"
+              route:
+                cluster: local_service
+        http_filters:
+        - name: envoy.health_check
+          config:
+           pass_through_mode: true
+           endpoint: "/healthcheck"
+           cache_time: 2.5s
+        - name: envoy.buffer
+          config:
+           max_request_bytes: 5242880
+           max_request_time: 0.12s
+        - name: envoy.router
+          config: {}
+        #Need to update access_log with logical_or and logical_and filters
+        # add calls to access_log_helper
+        access_log:
+         - name: envoy.file_access_log
+           filter:
+             not_health_check_filter:
+               {}
+           config:
+             path: "/var/log/envoy/egress_http_error.log"
+        idle_timeout: 0.84s
+        tracing:
+          operation_name: INGRESS
+{% endmacro -%}
+static_resources:
+  listeners:
+  {{ ingress_listener("tcp", "0.0.0.0", 9211) | indent(2)}}
+  #listener at 127.0.0.1:9001 is to be updated after access_log updates
+  #It is similar to listener at 127.0.0.1:9002
+  - name: listener_1
+    address:
+      socket_address:
+        protocol: TCP
+        port_value: 9002
+        address: 127.0.0.1
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          codec_type: AUTO
+          stat_prefix: egress_http
+          #change route_config to RDS
```

Comment:
instead of having route_config you would have something like:

```            
rds:
  config_source:
     api_config_source:
       api_type: GRPC
       cluster_names: rds
```

Reply:
and also add a rds cluster like in the v1 example